src/app/pages/account/groups/members/members.component.css

```diff
@@ -36,4 +36,5 @@
 
 mat-card-title {
   margin-left: 15px;
+  margin-top: 15px !important;
```

Comment:
You don't need `!important` here.
You can just increase [specificity of selector](https://specificity.keegan.st/), for example by using: `mat-card.form-card mat-card-title`.